src/redux/reducers/auth.js

```diff
@@ -0,0 +1,14 @@
+import { LOGIN_USER } from "../actions/actionTypes";
```

Comment:
this should be in the same folder as the login component and named `login.reducers.js`